docs/user-guide_installation.md

```diff
@@ -5,24 +5,26 @@ title: How to install DietPi
 
 # How to install DietPi
 
-The installation of DietPi consists of several steps:
+The installation of DietPi consists of few steps:
 
-1. Provide an installation media (e.g. SD card for single board computer or USB stick for PC)
-2. Get the DietPi image (and put it on the installation media)
-3. Boot up the DietPi device and go through one time installation steps
```

Comment:
I used a numbered list to show that the steps have to be conducted right in this order.
A bulleted list should be a list where the order is not relevant.

But a bulleted list is also o.k.

Reply:
Hi Stephan, 

Thanks a lot for the review. I suggested to make a bullet list in order to not puzzle the reader. On the same page we have also the installation steps (from 1 to 5). Let me know if this is ok.